lib/traject/config/sirsi_config.rb

```diff
@@ -1927,29 +1927,28 @@ def holdings(record, context)
   holdings(record, context).each do |holding|
     next if holding.skipped?
     next unless holding.gov_doc_loc? ||
+                holding.gov_docs_classification ||
                 marc_086.any? ||
                 holding.call_number_type == 'SUDOC'
 
     translation_map = Traject::TranslationMap.new('gov_docs_locations', default: 'Other')
     raw_location = translation_map[holding.home_location]
 
-    if raw_location == 'Other'
-      if marc_086.any?
-        marc_086.each do |marc_field|
-          gov_doc_values << if false && marc_field['2'] == 'cadocs'
-                              'California'
-                            elsif false && marc_field['2'] == 'sudocs'
-                              'Federal'
-                            elsif false && marc_field['2'] == 'undocs'
-                              'International'
-                            elsif marc_field.indicator1 == '0'
-                              'Federal'
-                            else
-                              raw_location
-                            end
-        end
-      else
-        gov_doc_values << raw_location
+    if holding.gov_docs_classification
+      gov_doc_values << holding.gov_docs_classification
+    elsif marc_086.any?
```

Comment:
Did this logic change? It seems like we dropped the outside condition?

```suggestion
    elsif raw_location == 'Other' && marc_086.any?
```